usaspending_api/etl/es_etl_helpers.py

```diff
@@ -406,6 +406,13 @@ def csv_chunk_gen(filename, chunksize, job_id, awards):
     dtype = {k: str for k in VIEW_COLUMNS if k not in converters}
     for file_df in pd.read_csv(filename, dtype=dtype, converters=converters, header=0, chunksize=chunksize):
         file_df = file_df.where(cond=(pd.notnull(file_df)), other=None)
+        if load_type != "awards":
```

Comment:
Should this instead check for the load_type equalling "transactions" instead in case we create new Indexes in the future?

Reply:
Does load type default to `transactions`? I thought it was null/empty if not set

Reply:
Per this bit from the es_rapidloader: 
```
parser.add_argument(
            "--load_type",
            type=str,
            help="Select which type of load to perform, current options are transactions or awards.",
            choices=["transactions", "awards"],
            default="transactions",
        )```

It should default to `transactions`.

Reply:
Fixed as ` == "transactions"`